### DISCUSSION TITLE
BMC5 Not starting on server side

### DISCUSSION THREAD

**Discussion by Ventilix** [itzg/docker-minecraft-server#3151] [Show and tell]
Hello,

Thanks again for the repo all contributors, extremely useful !!!!

I tryed to spin off a BMC5 (Better MineCraft 5 Server), it seems the very last mod (YUNG's Menu Tweaks (NeoForge)) is getting it to crash reboot as it is a client side mod not tagged correctly.
So you will have to add the mod exclusion (in compose add : CF_EXCLUDE_MODS: 1015152)
 
To help, here is a working compose as of today for me.
(it's a mix of various docker composes found in the repo not sure everything is useful but that worked for me ^^ on a ubuntu-22.04.4-live-server-amd64 LTS server using portainer)

services:
  mc:
    image: itzg/minecraft-server
    tty: true
    stdin_open: true
    restart: unless-stopped
    ports:
      - "22140:22140"
    environment:
      CF_FORCE_REINSTALL_MODLOADER: false
      EULA: "TRUE"
      LANG: en_US.UTF-8
      LANGUAGE: en_US:en
      LC_ALL: en_US.UTF-8
      JAVA_VERSION: jdk-21.0.4+7
      VERSION: 1.21.1
      UID: 1000
      GID: 1000
      MODPACK_PLATFORM: AUTO_CURSEFORGE
      CF_API_KEY: 'put your key here'
      CF_SLUG: better-mc-neoforge-bmc5
      ALLOW_FLIGHT: true
      MAX_MEMORY: 10G
      MOTD: "§l§cMessage of the day"
      MAX_PLAYERS: 15
      DIFFICULTY: hard
      MODE: survival
      SERVER_PORT: 22140
      VIEW_DISTANCE: 14
      SPAWN_PROTECTION: 0
      ONLINE_MODE: false
      SERVER_NAME: "SERVER_NAME"
      FETCH_TLS_HANDSHAKE_TIMEOUT: PT20S
      CF_EXCLUDE_MODS: 1015152
      SIMULATION_DISTANCE: 8
      ANNOUNCE_PLAYER_ACHIEVEMENTS: true
    volumes:
      # attach the relative directory 'data' to the container's /data path
      - /YOUR_PATH_HERE/MC_BMC5/data:/data

**Comment by Heat5ource** [comment]
Your post helped me add a bunch of QoL stuff to my servers and also help me troubleshoot my own stuff. I'll be sharing my own working config file for a unlisted example of stuff too. Thank you!

**Reply by Ventilix** [reply]
Glad it helped somebody, thx for sharing <3
